src/components/ProgramList.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import {
+    Box,
+    Heading,
+    Wrap,
+    WrapItem,
+    Badge,
+    Center,
+    AspectRatio,
+    Spacer,
+    Image,
+} from "@chakra-ui/react";
+
+export const ProgramList: React.FC = () => {
+    const handleClick = (event) => {
+        event.preventDefault();
+        console.log(event.target);
+        alert(
+            `clicked: ${
+                imagesAndDescriptions[event.target.getAttribute("data-index")]
+                    .title
+            }`,
+        );
+    };
+    const imagesAndDescriptions = [
+        {
+            image: "https://i.imgur.com/UaEscmK.jpeg",
+            title: "Building Bridges with Music",
+            description:
+                "Children with special needs will be able to connect with the music teacher through an online video call to socialize and have fun while learning about music!",
+            time: "July 7 to August 28, 2021",
+            type: "online",
+            category: "art",
+        },
+        {
+            image: "https://i.kym-cdn.com/entries/icons/original/000/026/489/crying.jpg",
+            title: "MPM/JELIC",
+            description:
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+            time: "July 7 to August 28, 2021",
+            type: "blended",
+            category: "music",
+        },
+        {
+            image: "https://metro.co.uk/wp-content/uploads/2017/07/187144066.jpg?quality=90&strip=all&zoom=1&resize=644%2C428",
+            title: "Education Through Creativity",
+            description:
+                "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+            time: "July 7 to August 28, 2021",
+            type: "online",
+            category: "math",
+        },
```

Comment:
we can turn this into a type or use an existing one

Reply:
where are the existing types :O

Reply:
there are some models in `/models` - I'm not sure if they fit here but if they don't we can just create a new type for this

Reply:
i made a new type (but why does the pr now have so many files changed after i rebased??)